README.md

```diff
@@ -218,6 +222,9 @@ See [Documentation/Configuration.md](Documentation/Configuration.md) for a
 description of the configuration file format and the settings that are
 available.
 
+Note that the contents of any `.swift-format-ignore`  are ignored; the
+file can be empty.
```

Comment:
Thinking about this further, we may eventually want to allow users to specify patterns here like other tools' ignore files. So we should make sure that the behavior that we support today would upgrade cleanly to something more elaborate in the future. If we allow the file to be empty today and then decide that we want to allow patterns, that becomes an awkward path because an empty file ought to mean "ignore nothing" in that world.

The most obvious thing that comes to mind is for us to currently require that the contents of the `.swift-format-ignore` file are exactly `*` (modulo whitespace). Then anyone who wants to ignore a directory could put that in their file and it would continue to work later if we start expanding it to more complex patterns. When we find the file, we could open it and error out if the file contains anything other than `*`, telling users that that's the only supported option today.

So I would remove this paragraph and change the one you added further up to read something like this instead:

> If it finds a file named `.swift-format-ignore`, its contents will determine which files in that directory should be ignored by swift-format. Currently, the only supported option is `*` (ignores all files).